src/query/useWasmGitHubAttestation.ts

```diff
@@ -0,0 +1,158 @@
+import { useQuery } from "@tanstack/react-query";
+import { rpc as StellarRpc } from "@stellar/stellar-sdk";
+import { isEmptyObject } from "@/helpers/isEmptyObject";
+import { AnyObject, NetworkHeaders } from "@/types/types";
+
+export const useWasmGitHubAttestation = ({
+  wasmHash,
+  sourceCodeLink,
+  rpcUrl,
+  isActive,
+  headers = {},
+}: {
+  wasmHash: string;
+  sourceCodeLink: string;
+  rpcUrl: string;
+  isActive: boolean;
+  headers?: NetworkHeaders;
+}) => {
+  const query = useQuery({
+    queryKey: ["useWasmGitHubAttestation", wasmHash, rpcUrl],
+    queryFn: async () => {
+      if (!wasmHash || !rpcUrl) {
+        return null;
+      }
+
+      try {
+        const rpcServer = new StellarRpc.Server(rpcUrl, {
+          headers: isEmptyObject(headers) ? undefined : { ...headers },
+          allowHttp: new URL(rpcUrl).hostname === "localhost",
+        });
+
+        const wasm = await rpcServer.getContractWasmByHash(wasmHash, "hex");
+        const sourceRepo = await extractSourceRepo(wasm);
+
+        const att = await fetch(
+          `https://api.github.com/repos/${sourceRepo}/attestations/sha256:${wasmHash}`,
```

Comment:
I think we should display this attestation path either as an URL so the user could click out to see, or display the attestation path as an URL and display the JSON too.

Reply:
That's a good idea. :+1: I added an attestation link that the user can open to view the JSON response.